src/Whoops/Resources/views/header.html.php

```diff
@@ -17,11 +17,16 @@
     COPY
   </button>
 
-  <p class="exc-message">
+  <div class="exc-message">
     <?php if (!empty($message)): ?>
       <?php echo $tpl->escape($message) ?>
     <?php else: ?>
       <span class="exc-message-empty-notice">No message</span>
     <?php endif ?>
-  </p>
+
+    <ul class="icon-list">
+      <li><a target="_blank" href="//google.com/search?q=<?php echo urlencode(implode('\\', $name).' '.$message) ?>" title="Search for help on Google."><i class="google"></i></a></li>
```

Comment:
I'd use explicit `https://`. No value in using non-secure link, especially when the exception message might be at least a little bit sensitive.